stdlib/2/xmlrpclib.pyi

```diff
@@ -12,7 +12,7 @@ from gzip import GzipFile
 _Unmarshaller = Any
 _timeTuple = Tuple[int, int, int, int, int, int, int, int, int]
 # Represents types that can be compared against a DateTime object
-_dateTimeComp = Union[AnyStr, DateTime, datetime, _timeTuple]
```

Comment:
These methods also accept objects that have a `timetuple` attribute, but not a tuple itself. This could be solved using a protocol, but I'll leave that to someone else.